integration/test/Test/Conversation.hs

```diff
@@ -44,7 +44,7 @@ testDynamicBackendsNotFederating = do
         $ bindResponse
           (API.getFederationStatus uidA [domainB, domainC])
         $ \resp -> do
-          resp.status `shouldMatchInt` 400
+          resp.status `shouldMatchInt` 422
```

Comment:
why do we have to change error codes all the time?  is it because moving to servant?

also, why wasn't that caught on concourse?

Reply:
I don't know why the code changed. These tests aren't running on CI.